zebra/main.c

```diff
@@ -139,9 +139,15 @@ static void sigint(void)
 
 	frr_early_fini();
 
-	list_delete_all_node(zebrad.client_list);
+	/*
```

Comment:
Didn't @qlyoung just make a change right here for the same type of thing for shutdown crashes?  Might want to look into it.

Reply:
I rebased the branch and removed the commit that modified this. I don't see the crash anymore with Quentin's fix.